R/pkg/R/column.R

```diff
@@ -91,6 +91,16 @@ createOperator <- function(op) {
             })
 }
 
+createInOperator <- function(op) {
```

Comment:
Actually we don't need the `createInOperator` function -- You can just add the `setMethod` below after `cast` etc.